tests/components/radarr/test_sensor.py

```diff
@@ -192,182 +189,211 @@ def json(self):
     return MockResponse({"error": "Unauthorized"}, 401)
 
 
-class TestRadarrSetup(unittest.TestCase):
-    """Test the Radarr platform."""
-
-    # pylint: disable=invalid-name
-    DEVICES = []
-
-    def add_entities(self, devices, update):
-        """Mock add devices."""
-        for device in devices:
-            self.DEVICES.append(device)
-
-    def setUp(self):
-        """Initialize values for this testcase class."""
-        self.DEVICES = []
-        self.hass = get_test_home_assistant()
-        self.hass.config.time_zone = "America/Los_Angeles"
-        self.addCleanup(self.tear_down_cleanup)
-
-    def tear_down_cleanup(self):
-        """Stop everything that was started."""
-        self.hass.stop()
-
-    @patch("requests.get", side_effect=mocked_requests_get)
-    def test_diskspace_no_paths(self, req_mock):
-        """Test getting all disk space."""
-        config = {
+async def test_diskspace_no_paths(hass):
+    """Test getting all disk space."""
+    config = {
+        "sensor": {
             "platform": "radarr",
             "api_key": "foo",
             "days": "2",
             "unit": DATA_GIGABYTES,
             "include_paths": [],
             "monitored_conditions": ["diskspace"],
         }
-        radarr.setup_platform(self.hass, config, self.add_entities, None)
-        for device in self.DEVICES:
-            device.update()
-            assert "263.10" == device.state
-            assert "mdi:harddisk" == device.icon
-            assert DATA_GIGABYTES == device.unit_of_measurement
-            assert "Radarr Disk Space" == device.name
-            assert "263.10/465.42GB (56.53%)" == device.device_state_attributes["/data"]
-
-    @patch("requests.get", side_effect=mocked_requests_get)
-    def test_diskspace_paths(self, req_mock):
-        """Test getting diskspace for included paths."""
-        config = {
+    }
+
+    with patch(
+        "requests.get",
+        side_effect=mocked_requests_get,
+    ):
+        assert await async_setup_component(hass, "sensor", config)
+        await hass.async_block_till_done()
+
+        entity = hass.states.get("sensor.radarr_disk_space")
+        assert entity is not None
+        assert "263.10" == entity.state
```

Comment:
Please put the expected value on the right of the `==`
```suggestion
        assert entity.state == "263.10"
```